runtime/gc_vlhgc/CopyForwardScheme.hpp

```diff
@@ -1093,6 +1102,24 @@ class MM_CopyForwardScheme : public MM_BaseNonVirtual
 		_regionCountReservedNonEvacuated = regionCount;
 	}
 
+#if defined(OMR_GC_VLHGC_CONCURRENT_COPY_FORWARD)
+	/**
+	 * Run concurrent copy forward collection increment
+	 * @param env[in] Main thread.
+	 */
+	void concurrentCopyForwardCollectionSet(MM_EnvironmentVLHGC *env);
+
+	/**
+	 * True if concurrent CopyForward cycle is active at any point (STW or concurrent
+	 * task active, or even short gaps between STW and concurrent tasks). Equivalent to
+	 * isConcurrentCycleInProgress() from Scavenger
+	 */
+	MMINLINE bool isConcurrentCycleInProgress() {
```

Comment:
I suspect this method will be frequently used, and for the sake of simpler call sites, I'd make it always (compile time) availabe. The internal implementaion, then can hard code 'return false' if the build flag is off (and return proper runtime flag when it's on)

Reply:
BTW, this is alraedy the case for GCExt::isConcurrentCopyForwardEnabled() in OMR.

Reply:
I see that makes sense!